etc/config/analysis.amazon.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&llvm-mca
```

Comment:
This file adds a new language but it's not defined anywhere else (Proably `libs/languajes.js`)

Reply:
Added.
I also added an example which tells the user to add the -mcpu=... argument, just so no one gets stuck there :-)